### PR TITLE
fix numpy embeddings errors

### PR DESCRIPTION
**Pull request by ArtemVazh**

### CODE REVIEW

File: src/lm_polygraph/estimators/mahalanobis_distance.py [hunk 1]
```diff
@@ -120,18 +120,20 @@ def __call__(self, stats: Dict[str, np.ndarray]) -> np.ndarray:
 
         # take the embeddings
         embeddings = stats[f'embeddings_{self.embeddings_type}']  
+        if not isinstance(embeddings, torch.Tensor):
+            embeddings = torch.tensor(embeddings)
```
Comment (rvashurin) [inline src/lm_polygraph/estimators/mahalanobis_distance.py:124]: torch.from_numpy() может лучше? или .tensor сам разберется?

File: src/lm_polygraph/estimators/mahalanobis_distance.py [hunk 2]
```diff
@@ -120,18 +120,20 @@ def __call__(self, stats: Dict[str, np.ndarray]) -> np.ndarray:
...
 
         # compute centroids if not given     
         if self.centroid is None:
-            self.centroid = stats[f'train_embeddings_{self.embeddings_type}'].mean(axis=0)
+            self.centroid = torch.tensor(stats[f'train_embeddings_{self.embeddings_type}'].mean(axis=0))
```
Comment (rvashurin) [inline src/lm_polygraph/estimators/mahalanobis_distance.py:128]: а можем просто embeddings.mean(dim=0)? кстати axis vs dim, торч и то и то понимает?

File: src/lm_polygraph/estimators/mahalanobis_distance.py (hunk 1 above)
Comment (ArtemVazh) [reply src/lm_polygraph/estimators/mahalanobis_distance.py:124]: Да, наверное, лучше from_numpy, т.к. тогда тип данных сохраняется, в .tensor автоматом к FloatTensor

File: src/lm_polygraph/estimators/mahalanobis_distance.py (hunk 2 above)
Comment (ArtemVazh) [reply src/lm_polygraph/estimators/mahalanobis_distance.py:128]: Сейчас среднее в numpy и потом в тензор, могу поменять, но не думаю, что есть большая разница

Comment (rvashurin) [reply src/lm_polygraph/estimators/mahalanobis_distance.py]: А все забей я прочитал плохо

**Review by rvashurin — CHANGES_REQUESTED** [review]

**Review by rvashurin — APPROVED** [review]
